### ISSUE TITLE
Add pdb / set_trace() check

### ISSUE THREAD

**Issue by giampaolo** [astral-sh/ruff#7793]
Basically this: https://pypi.org/project/flake8-debug/. Right now ruff is able to identify the `breakpoint` builtin function, which results in:

```
setup.py:29:13: T100 Trace found: `breakpoint` used
```

`breakpoint` builtin is a relatively new addition though (python 3.7, see [doc](https://docs.python.org/3/library/functions.html#breakpoint)). A lot of people (myself included) still use the `pdb` module, and may accidentally commit code like this:

```python
import pdb; pdb.set_trace()
```

This is particularly dangerous if the above code gets accidentally deployed in production.

**Comment by zanieb** [comment]
Makes sense to me!

**Comment by tjkuson** [comment]
I believe the rule `debugger` already catches this case.

```python
import pdb; pdb.set_trace()
```

triggers the rule on my machine.

**Comment by zanieb** [comment]
Can confirm that `T100` is raised here `example.py:1:13: T100 Trace found: `pdb.set_trace` used`

**Comment by giampaolo** [comment]
Whops! I must have done something wrong in my testiong since I also confirm `pdb.set_trace` results in T100 being raised. Sorry about the noise. :) Closing this out.
